docs-ref-conceptual/install-azure-cli-apt.md

```diff
@@ -24,15 +24,13 @@ for the Azure CLI. This package has been tested with:
 1. <a name="install-step-1"/> Modify your sources list:
 
     ```bash
-    AZ_REPO=$(lsb_release -cs)
```

Comment:
This removed variable is referenced later in troubleshooting and exists so that users can manually set it if necessary. Needs to be added back.